reporter-web-app/build.gradle

```diff
@@ -1,43 +1,33 @@
-plugins {
-    id 'com.moowork.node' version '1.2.0'
+import org.apache.tools.ant.taskdefs.condition.Os
```

Comment:
In `GettingStarted.md` no WebApp report is created. Do you still think Yarn should be added as a prerequisite there?

About the version check, we do in fact not require any specific versions for the WebApp. With the previous plugin we just hard-coded the versions so no other versions than the already installed ones get bootstrapped, AFAIK.

Reply:
Yes, because the build of `cli` depends on building the web app. And we will eventually switch to creating the web app report in the guide.
At least in some environments it seems the Yarn version is important, but maybe that was specific to AppVeyor: 431a053
I think we should at least provide a version which is verified to work, just in case a new release breaks the build.

Reply:
Yes, that was specific to AppVeyor, we simply must not use another version than the one we already have on AppVeyor. In that regard the dropped bootstrapping is even a good thing 😉